.github/workflows/build-devcontainer-image.yml

```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   build-and-push-image:
-    if: github.event.pull_request.merged == true
+    if: github.repository == 'cds-snc/cra-arc-upd-tbpc' && github.event.pull_request.merged == true
```

Comment:
Just curious why you are specifying the repo here since it should by default this action will execute in this repo only.

Reply:
I see. I did not think that you will be working off the parent repo and just off of this one.